wasm_bindgen/raze/remote/mime-0.2.6.BUILD

```diff
@@ -36,6 +36,6 @@ rust_library(
     ],
     version = "0.2.6",
     deps = [
-        "@raze__log__0_3_9//:log",
+        "@raze__log__0_4_8//:log",
```

Comment:
I do not know why, but tests were failing for me until I changed this as well. Seems safe to me though.

Reply:
Can't you just force the log version in the Cargo.toml file?